projects/plugins/super-cache/wp-cache.php

```diff
@@ -2612,7 +2613,7 @@ function wp_cache_files() {
 	} else {
 		echo "<p>" . __( 'Cache stats are not automatically generated. You must click the link below to regenerate the stats on this page.', 'wp-super-cache' ) . "</p>";
 		echo "<a href='" . wp_nonce_url( add_query_arg( array( 'page' => 'wpsupercache', 'tab' => 'contents', 'action' => 'regenerate_cache_stats' ) ), 'wp-cache' ) . "'>" . __( 'Regenerate cache stats', 'wp-super-cache' ) . "</a>";
-		if ( is_array( $cache_stats ) ) {
+		if ( $cache_stats ) {
```

Comment:
Behavior change: before an empty array would still run line 2617, now it won't.

And the "now it won't" isn't sufficient to avoid the possibility that `$cache_stats[ 'generated' ]` is unset.